core/stats.go

```diff
@@ -33,18 +33,18 @@ func (s *Stats) Update(p publisher) {
 	}
 
 	stats := map[string]map[string]float64{
-		"30d":      s.calculate(30),
-		"365d":     s.calculate(365),
-		"thisYear": s.calculate(time.Now().YearDay()),
-		"total":    s.calculate(365 * 100), // 100 years
+		"30d":      s.calculate(time.Now().AddDate(0, 0, -30)),
+		"365d":     s.calculate(time.Now().AddDate(0, 0, -365)),
+		"thisYear": s.calculate(time.Date(time.Now().Year(), 1, 1, 0, 0, 0, 0, time.Local)),
```

Comment:
Alternativ:

    now.BeginningOfYear()

in der gleichen Art wäre auch "this month" möglich.

Reply:
uhh, nice!